### PR TITLE
ISPN-11623: use default to connect to a server without arguments

### PR DESCRIPTION
**Pull request by wfink**
Simplify connected start without parameter for -c --connect
https://issues.redhat.com/browse/ISPN-11623

### CODE REVIEW

File: cli/cli-client/src/main/java/org/infinispan/cli/CLI.java [hunk 1]
```diff
@@ -108,10 +108,15 @@ public final void run(String[] args) {
          }
          switch (command) {
             case "-c":
-               parameter = iterator.next();
+               if(iterator.hasNext()) {
```
Comment (tristantarrant) [inline cli/cli-client/src/main/java/org/infinispan/cli/CLI.java:111]: Not quite: the next element could be another switch.

Comment (wfink) [reply cli/cli-client/src/main/java/org/infinispan/cli/CLI.java:111]: This is the same for other options (and server.sh as well)
If the parameter is missed the error message might become confusing

File: cli/cli-client/src/main/java/org/infinispan/cli/CLI.java [hunk 2]
```diff
@@ -108,10 +108,15 @@ public final void run(String[] args) {
...
+                   parameter = iterator.next();
+               }
                // Fall through
             case "--connect":
                connectionString = parameter;
+               if(connectionString == null) {
+                   connectionString="localhost:11222";
```
Comment (tristantarrant) [inline cli/cli-client/src/main/java/org/infinispan/cli/CLI.java:118]: This should be `http://localhost:11222`. Also please fix the formatting.

File: cli/cli-client/src/main/java/org/infinispan/cli/CLI.java (hunk 1 above)
Comment (tristantarrant) [reply cli/cli-client/src/main/java/org/infinispan/cli/CLI.java:111]: No, because I want this to work for something like `-c -f file`

**Comment by tristantarrant** [comment]
This functionality has been absorbed into https://github.com/infinispan/infinispan/pull/8295

**Pull request closed without merging**